public/scripts/services/requestService.js

```diff
@@ -237,10 +237,9 @@ devflowApp.factory('requestService', ['$http', 'socket', 'notificationService',
 
         release: function(request) {
             request.state = 'posted';
-            request.reviewer = null;
             request.taken_date = null;
             request.sort_order = 1;
-            socket.emit('updateRequest', request);
```

Comment:
No since I replaced it with a different action called releaseRequest which does the same plus some more.